### PR TITLE
feat(dropdowns): Add CSS vars for $dropdown

### PR DESCRIPTION
**Pull request by isabellechanclou**
Add CSS vars for Dropdowns components.

### CODE REVIEW

File: scss/_dropdown.scss [hunk 1]
```diff
@@ -9,27 +9,36 @@
 }
 
 .dropdown-toggle {
```
Comment (louismaximepiton) [inline scss/_dropdown.scss:11]: IMO, the `.dropdown-toggle` should stay as the Bootstrap one, and IMO we should introduce a `.btn-dropdown` but in another PR ! (maybe #1164 if accepted)

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Almost there, 3 few things to tackle

File: site/content/docs/5.1/components/dropdowns.md [hunk 2]
```diff
@@ -853,10 +853,17 @@ By default, the dropdown menu is closed when clicking inside or outside the drop
...
 
+Variables for all dropdowns:
 {{< scss-docs name="dropdown-css-vars" file="scss/_dropdown.scss" >}}
 
-Customization through CSS variables can be seen on the `.dropdown-menu-dark` class where we override specific values without adding duplicate CSS selectors.
+Customization through CSS variables can be seen on the following classes where we override specific values without adding duplicate CSS selectors:
+- on the `.dropdown-toggle` class:
+{{< scss-docs name="dropdown-toggle-css-vars" file="scss/_dropdown.scss" >}}
+
+- on the `.dropdown-item` class:
```
[the remark this answers is not in the document]
Comment (isabellechanclou) [reply site/content/docs/5.1/components/dropdowns.md:863]: OK

File: scss/_dropdown.scss [hunk 3]
```diff
@@ -71,6 +71,7 @@
   --#{$prefix}dropdown-link-disabled-color: #{$dropdown-link-disabled-color};
   --#{$prefix}dropdown-item-padding-x: #{$dropdown-item-padding-x};
   --#{$prefix}dropdown-item-padding-y: #{$dropdown-item-padding-y};
+  --#{$prefix}dropdown-item-font-weight-bold: #{$font-weight-bold}; // Boosted mod
```
Comment (louismaximepiton) [inline scss/_dropdown.scss:74]: Should be `--#{$prefix}dropdown-item-font-weight`

File: site/content/docs/5.1/components/dropdowns.md [hunk 4]
```diff
@@ -854,15 +854,13 @@ By default, the dropdown menu is closed when clicking inside or outside the drop
 As part of Boosted's evolving CSS variables approach, dropdowns now use local CSS variables on `.dropdown-menu` for enhanced real-time customization. Values for the CSS variables are set via Sass, so Sass customization is still supported, too.
 
 Variables for all dropdowns:
+
 {{< scss-docs name="dropdown-css-vars" file="scss/_dropdown.scss" >}}
 
 Customization through CSS variables can be seen on the following classes where we override specific values without adding duplicate CSS selectors:
 - on the `.dropdown-toggle` class:
```
Comment (louismaximepiton) [inline site/content/docs/5.1/components/dropdowns.md:862]: Should add a line between text and shortcodes (same few lines below)

File: site/content/docs/5.1/components/dropdowns.md [hunk 5]
```diff
@@ -889,6 +890,7 @@ Mixins are used to generate the CSS-based carets and can be found in `scss/mixin
 Via data attributes or JavaScript, the dropdown plugin toggles hidden content (dropdown menus) by toggling the `.show` class on the parent `.dropdown-menu`. The `data-bs-toggle="dropdown"` attribute is relied on for closing dropdown menus at an application level, so it's a good idea to always use it.
 
 {{< callout info >}}
+
```
Comment (louismaximepiton) [inline site/content/docs/5.1/components/dropdowns.md:893]: This line should be removed to be closer to Bs

File: site/content/docs/5.1/components/dropdowns.md [hunk 6]
```diff
@@ -254,6 +254,7 @@ And putting it to use in a navbar:
 
 {{< callout info >}}
 #### RTL
+
```
Comment (louismaximepiton) [inline site/content/docs/5.1/components/dropdowns.md:257]: This line should be removed to be closer to Bs

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
I think we had a small misunderstanding with the last commit

**Review by MewenLeHo — APPROVED** [review]
LGTM 👌

**Pull request merged**